### ISSUE TITLE
Dependency failing in GitHub due to bintray.com turn down

### ISSUE THREAD

**Issue by shanemc-goog** [seattle-uat/civiform#2250]
Most GitHub actions are failing, looks like bintray.com is expected by a dependency and that site has been turned down it seems

https://stackoverflow.com/questions/70677130/bintray-502-bad-gateway-in-android-studio

```
[warn] 	Note: Unresolved dependencies path:
Error:  sbt.librarymanagement.ResolveException: Error downloading org.webjars.npm:mime-db:[1.52.0]
Error:    No version found for [1.52.0,1.52.0] in 
Error:    No version found for [1.52.0,1.52.0] in 
Error:    not found: /root/.ivy2/localorg.webjars.npm/mime-db
Error:    No version found for [1.52.0,1.52.0] in https://repo1.maven.org/maven2/org/webjars/npm/mime-db/maven-metadata.xml
Error:    download error: Caught java.io.IOException (Server returned HTTP response code: 502 for URL: https://dl.bintray.com/webjars/maven/org/webjars/npm/mime-db/maven-metadata.xml) while downloading https://dl.bintray.com/webjars/maven/org/webjars/npm/mime-db/maven-metadata.xml
Error:    not found: https://build.shibboleth.net/nexus/content/groups/public/org/webjars/npm/mime-db/maven-metadata.xml
Error:  	at lmcoursier.CoursierDependencyResolution.unresolvedWarningOrThrow(CoursierDependencyResolution.scala:345)
Error:  	at lmcoursier.CoursierDependencyResolution.$anonfun$update$38(CoursierDependencyResolution.scala:314)
Error:  	at scala.util.Either$LeftProjection.map(Either.scala:[57](https://github.com/seattle-uat/civiform/runs/5807765913?check_suite_focus=true#step:5:57)3)
Error:  	at lmcoursier.CoursierDependencyResolution.update(CoursierDependencyResolution.scala:314)
Error:  	at sbt.librarymanagement.DependencyResolution.update(DependencyResolution.scala:60)
Error:  	at sbt.internal.LibraryManagement$.resolve$1(LibraryManagement.scala:[59](https://github.com/seattle-uat/civiform/runs/5807765913?check_suite_focus=true#step:5:59))
Error:  	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$12(LibraryManagement.scala:133)
Error:  	at sbt.util.Tracked$.$anonfun$lastOutput$1(Tracked.scala:73)
Error:  	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$20(LibraryManagement.scala:146)
Error:  	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
Error:  	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11(LibraryManagement.scala:146)
Error:  	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11$adapted(LibraryManagement.scala:127)
Error:  	at sbt.util.Tracked$.$anonfun$inputChangedW$1(Tracked.scala:219)
Error:  	at sbt.internal.LibraryManagement$.cachedUpdate(LibraryManagement.scala:1[60](https://github.com/seattle-uat/civiform/runs/5807765913?check_suite_focus=true#step:5:60))
Error:  	at sbt.Classpaths$.$anonfun$updateTask0$1(Defaults.scala:3690)
Error:  	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
Error:  	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
Error:  	at sbt.std.Transform$$anon$4.work(Transform.scala:68)
Error:  	at sbt.Execute.$anonfun$submit$2(Execute.scala:282)
Error:  	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
Error:  	at sbt.Execute.work(Execute.scala:291)
Error:  	at sbt.Execute.$anonfun$submit$1(Execute.scala:282)
Error:  	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
Error:  	at sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
Error:  	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
Error:  	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
Error:  	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
Error:  	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
Error:  	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
Error:  	at java.base/java.lang.Thread.run(Thread.java:834)
Error:  (update) sbt.librarymanagement.ResolveException: Error downloading org.webjars.npm:mime-db:[1.52.0]
Error:    No version found for [1.52.0,1.52.0] in 
Error:    No version found for [1.52.0,1.52.0] in 
Error:    not found: /root/.ivy2/localorg.webjars.npm/mime-db
Error:    No version found for [1.52.0,1.52.0] in https://repo1.maven.org/maven2/org/webjars/npm/mime-db/maven-metadata.xml
Error:    download error: Caught java.io.IOException (Server returned HTTP response code: 502 for URL: https://dl.bintray.com/webjars/maven/org/webjars/npm/mime-db/maven-metadata.xml) while downloading https://dl.bintray.com/webjars/maven/org/webjars/npm/mime-db/maven-metadata.xml
Error:    not found: https://build.shibboleth.net/nexus/content/groups/public/org/webjars/npm/mime-db/maven-metadata.xml
Error:  Total time: 25 s, completed Apr 3, 2022, 7:53:00 PM
Error: Process completed with exit code 1.
```

**Comment by shanemc-goog** [comment]
Susan and I paired on this.

The cause and therefor resolution was inconclusive but it seems to have passed.

Between us, Gwen and Swathi; only me and GitHub actively showed the issue.

Shane: I was only able to repo in a single local repo.  Checking the repo out in a new location did not have the issue. I was able to resolve the issue seemingly my removing my ~/.ivy directory.   Oddly moving it back did not recreate the problem.  I'm passed it seemingly

GityHub: GH started inconsistently showing the issue.  I reran the prechecks on various PRs, some gave the issue others did not, and it seems like they're passing now consistently.


One outstanding thought we had but can't speak towards yet, is if somehow my local corrupted state pushed something to DockerHub/etc that the GitHub system then picked up.
